src/app/kontakt/components/LegalInfo.tsx

```diff
@@ -0,0 +1,43 @@
+// LegalInfo.js
+import IconBriefcase from "@/icons/IconBriefcase";
+import IconBook from "@/icons/IconBook";
+import IconDollar from "@/icons/IconDollar";
+
+export default function LegalInfo() {
+    return (
+        <section aria-labelledby="informacje prawne" className="space-y-4 text-left max-w-[260px] mt-8 tablet:mt-0">
+            {/* NIP */}
+            <div className="flex gap-5 items-center desktop:gap-8 desktop:mb-8">
+                <div className="w-6 flex-shrink-0">
+                    <IconBriefcase name="briefcase" className="w-8 h-8" />
+                </div>
+                <div>
+                    <p className="desktop:mb-4">NIP:</p>
+                    <p className="font-medium">5261710045</p>
+                </div>
+            </div>
+
+            {/* KRS */}
+            <div className="flex items-center gap-5 desktop:gap-8 desktop:mb-8 tablet:hidden desktop:flex">
```

Comment:
Why are we hiding in on the tablet? Makes no sense.
If the design has some potential mistakes then just contact the design team 😉 

We all make mistakes and we should collaborate together to ensure the quality of the product - which means that communication plays an important role in the product/software development.
If something is not clear - just ask the proper people 😄

Personally I treat design the same way the pirates treat the Pirate Code - a set of guidelines not rules 😉 
https://www.youtube.com/watch?v=k9ojK9Q_ARE